broker/aws/impl.go

```diff
@@ -387,6 +388,10 @@ func (b *Broker) getAWSRolesForAccount(accountName string) ([]string, error) {
 			b.logger.Debugf(1, "Got roles from cache")
 			return cachedEntry.Roles, nil
 		}
+		if cachedEntry.LastBadTime.After(time.Now().Add(time.Second * (-1 * negativeCacheSeconds))) {
```

Comment:
You can just do `-negativeCacheSeconds`.

Reply:
done